schemas/soknad/common-fields.ts

```diff
@@ -66,6 +66,8 @@ export const alertTextField = {
   type: "object",
   name: "alertText",
   title: "Varseltekst",
+  //@ts-ignore;
```

Comment:
lgtm, hva klager ts-ignoren på?

Reply:
Det finnes ikke en interface vi kan bruke fra sanity-types, siden `document.aktivateAlertText` er det noe vi har definert selv. Alternative er interface selv, har vi lyst å gjøre det for å droppe //@ts-ignore ?